docs/pages/connect-your-client/tsh.mdx

```diff
@@ -599,6 +599,9 @@ scp"` and use the familiar syntax:
 $ scp -P 61122 -r files root@node:/path/to/dest
 ```
 
+Note that `tsh scp` always uses the SFTP protocol under the hood, no flags need
```

Comment:
```suggestion
Note that `tsh scp` always uses the SFTP protocol under the hood, so no flags need
```

Reply:
I'd add that `tsh scp` uses SFTP starting from Teleport 11